src/vpp_plugins/vpp-chaperone/VppConfigManager.cpp

```diff
@@ -1644,89 +1650,98 @@ void VppConfigManager::doCpePolicerConfig (VppClient &vppClient,
   // otherwise VPP will crash if policed traffic is being passed while the
   // config is deleted.
   bool ifaceStopped = false;
-  // Delete old tables
-  std::vector<u32> tableIds;
-  vppClient.getClassifierTableIds (tableIds);
   VppClient::ClassifierTableConfig tableConfig;
-  for (const u32 &tableId : tableIds)
+  u32 tableId = ~0;
+
+  vppClient.getClassifierByIndex (interfaceIndex, tableId);
+  if (tableId != ~0)
     {
       if (!ifaceStopped)
         {
           ifaceStopped = true;
-          VLOG (1) << "Stopping interface "
-                   << interface << " before removing old policer config";
+          VLOG (1) << "Stopping interface " << interface
+                   << " before removing old policer config";
           vppClient.setInterfaceFlags (interface, false);
         }
       tableConfig.tableIndex = tableId;
       tableConfig.isAdd = 0; // Delete old table
       vppClient.addDelClassifierTable (tableConfig);
     }
+  else
+    {
+      LOG (ERROR) << "Interface does not have classifier table";
+    }
```

Comment:
This should not use `LOG (ERROR)` since it's not an error if an interface does not have a classifier table in this case.